helix-term/Cargo.toml

```diff
@@ -35,6 +35,7 @@ helix-loader = { path = "../helix-loader" }
 anyhow = "1"
 once_cell = "1.20"
 
+heck = "0.5"
```

Comment:
Looks very nice, I think this is the only thing that maintainers might want to consider, since it's adding a new dependency. Otherwise I am very impressed with how small the changes are! Congrats!

Reply:
Thanks! I'm gonna use Helix's built-in case conversions so the dependency is removed